scanpy/tools/_score_genes.py

```diff
@@ -79,7 +79,7 @@ def score_genes(
             genes_to_ignore.append(gene)
     if len(genes_to_ignore) > 0:
         logg.warning(f'genes are not in var_names and ignored: {genes_to_ignore}')
-    gene_list = set(gene_list_in_var[:])
```

Comment:
Why slice this? `set` isn’t a view class.

Reply:
I think that I was trying to get rid of duplicate gene names.

Reply:
No, the question is: why `set(x[:])` and not `set(x)`?